target/linux/generic/pending-6.6/778-net-l2tp-drop-flow-hash-on-forward.patch

```diff
@@ -19,9 +19,9 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
 
 --- a/net/l2tp/l2tp_eth.c
 +++ b/net/l2tp/l2tp_eth.c
-@@ -136,6 +136,9 @@ static void l2tp_eth_dev_recv(struct l2t
- 	/* checksums verified by L2TP */
- 	skb->ip_summed = CHECKSUM_NONE;
+@@ -139,6 +139,9 @@ static void l2tp_eth_dev_recv(struct l2t
+ 	/* drop outer flow-hash */
+ 	skb_clear_hash(skb);
  
 +	/* drop outer flow-hash */
 +	skb_clear_hash(skb);
```

Comment:
It seems that this patch has been upstreamed. Please check it again.

Reply:
Interesting that the script did not find this.  I ran it again as a test.  Did you identify this manually or did your script identify this?

Reply:
No script, just some little experience haha~